proto/ssz_proto_library.bzl

```diff
@@ -42,6 +42,7 @@ mainnet = {
     "bytes_per_cell.size": "2048",  # FIELD_ELEMENTS_PER_CELL * BYTES_PER_FIELD_ELEMENT
     "cells_per_blob.size": "128",
     "kzg_commitments_inclusion_proof_depth.size": "4",
+    "proposer_lookahead_size": "64",  # (MIN_SEED_LOOKAHEAD + 1) * SLOTS_PER_EPOCH
```

Comment:
Why do you need this if it's not different between mainnet/minimal? Can't it be hardcoded constant in the proto?

Reply:
Discussed offline, I was mistaken